### ISSUE TITLE
Trigger MIDI Channel setting not working.

### ISSUE THREAD

**Issue by Stazed** [lsp-plugins/lsp-plugins#411]
Affects all plugin types.

Looks like missing:
```
nChannel    = pChannel->value();
```
In:
```
/home/sspresto/Programming/lsp-plugins/modules/lsp-plugins-trigger/src/main/plug/trigger.cpp
```
```
void trigger::update_settings()
```
Adding above fixed it for me.

**Comment by sadko4u** [comment]
Seems that you found a very rare bug :). Thanks.

**Comment by sadko4u** [comment]
Pushed changes to `devel`.

**Comment by sadko4u** [comment]
Fixed in 1.2.17.
